lib/bolt/cli/src/commands/admin/cluster/datacenter.rs

```diff
@@ -71,6 +88,29 @@ pub enum SubCommand {
 		#[clap(long, short = 'd')]
 		datacenter_name_id: String,
 	},
+	Update {
```

Comment:
nit: rename to `UpdatePool <dc> <pool_type>` followed by the rest of the flags. wehre dc & pool_type is positional

Reply:
In this case, should cluster also be positional?